include/umf/memspace.h

```diff
@@ -44,6 +44,12 @@ umfMemoryProviderCreateFromMemspace(umf_memspace_handle_t hMemspace,
                                     umf_memspace_policy_handle_t hPolicy,
                                     umf_memory_provider_handle_t *hProvider);
 
+///
+/// \brief Retrieves predefined host memspace.
+/// \return UMF_RESULT_SUCCESS on success or appropriate error code on failure.
+///
+umf_result_t umfMemspaceHostGet(umf_memspace_handle_t *hMemspace);
```

Comment:
Are we using getter function instead of predefined global memspace variable for the same reason (to be able to build UMF as a static lib) we replaced global OPS structures for pools/providers?

Reply:
Yeah, @igchor mentioned that it would be for the best if host memspace followed the same approach as with ops.